pkg/ctl/client.go

```diff
@@ -70,6 +70,20 @@ func (dkvClnt *DKVClient) Put(key []byte, value []byte) error {
 	return errorFromStatus(status, err)
 }
 
+// Delete takes the key as byte arrays and invokes the
+// GRPC Delete method. This is a convenience wrapper.
+func (dkvClnt *DKVClient) Delete(key []byte) error {
```

Comment:
Please include a test inside `internal/master/ss_service_test.go` for this functionality.

Reply:
Have added.